### ISSUE TITLE
Portal open/save paths broken at start

### ISSUE THREAD

**Issue by stefonarch** [lxqt/xdg-desktop-portal-lxqt#36]
It happened today after an update, no idea what component could be involved, it was mostly python. Broken both in firefox and thunderbird, while element (xwayland) doesn't use it anymore.

![immagine](https://github.com/lxqt/xdg-desktop-portal-lxqt/assets/10681413/7725d8cd-58f2-4bb8-b9e5-408585fe6fd4)

![immagine](https://github.com/lxqt/xdg-desktop-portal-lxqt/assets/10681413/8c935f56-b74d-4dbf-971b-5ac9f950637e)

Filename is also empty if trying to save something.

**Comment by tsujan** [comment]
It happened to me a few weeks ago, when I tried the portal for the fist time since its creation. The problem is in that `%00`, which I don't know where comes from. Here the problem disappeared after I removed some redundant keys about "dir" and "down" (those with trash icons beside them) from `about::config`, but it showed up again after a short time.

I also remember that there was no problem in Chromium (I uninstalled the portal after checking it).
